dali/test/python/test_operator_resize.py

```diff
@@ -527,3 +527,72 @@ def test_stitching():
                 for channel_first in [False, True]:
                     for interp in [types.INTERP_LINEAR, types.INTERP_CUBIC, types.INTERP_TRIANGULAR, types.INTERP_LANCZOS3]:
                         yield _test_stitching, device, dim, channel_first, dtype, interp
+
+def _test_empty_input(dim, device):
```

Comment:
You can use: 
`from nose.tools import nottest`
```suggestion
@nottest
def test_empty_input(dim, device):
```

to state the intention and not make one wander why it's private. Or I would just write `check_empty_input`.

Reply:
> `@nottest`
I can add `@nottest`, but I'd keep the underscore - `check_` functions have a different purpose in our tests (_vide_ `check_batch`). Still, that's just more typing and the pattern of public enumerator and private test implementation is rather pervasive in our tests.

> `check_empty_input`
I'd call that confusing - sounds too much like our `check_batch`.

Reply:
I guess it's anecdotal evidence, as I think it's first time I see functions starting with underscore and I rather see a lot of `check_something` to circumvent the nose running `test_something` functions. Whatever.